docs/basic_usage/gpt_oss.md

```diff
@@ -127,3 +127,231 @@ The sum of 29,138,749,187 and 29,138,749,187 is **58,277,498,374**.
 **Bottom line:** Nvidia’s stock is on a strong upward trajectory in 2025, with price targets climbing toward $200‑$210 and the market price already near $175 as of late August.
 
 ```
+
+# Quick Start Recipe for GPT-OSS on NVIDIA Blackwell Hardware
+
+This section provides step-by-step instructions for running the GPT-OSS-20b/120b model using SGLang with MXFP4 quantization, optimized for NVIDIA Blackwell architecture GPUs. It covers the complete setup required; from accessing model weights and preparing the software environment to configuring SGLang parameters, launching the server, and validating inference output.
+
+The recipe is intended for developers and practitioners seeking high-throughput or low-latency inference using NVIDIA’s accelerated stack—building a docker image with SGLang for model serving and FlashInfer for optimized CUDA kernels.
+
+## Prerequisites
+
+- OS: Linux
+- Drivers: CUDA Driver 575 or above
+- GPU: Blackwell architecture
+- [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/index.html)
+
+## Deployment Steps
+
+### Pull Docker Image
+
+Pull the SGLang v0.5.5.post1 release docker image.
```

Comment:
this should go to https://github.com/sgl-project/sglang/blob/main/docs/get_started/install.md and you only need to refer to https://docs.sglang.ai/get_started/install.html here

Reply:
Added an extra sentence:

> Please refer to the [SGLang installation guide](https://docs.sglang.ai/get_started/install.html) about how to install SGLang. In this guide, we assume using the official SGLang docker image.